File: app/src/main/java/com/example/stud_informer_irkpo/network/ScheduleApi.java
package com.example.stud_informer_irkpo.network;

import com.example.stud_informer_irkpo.models.ScheduleResponse;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;
public interface ScheduleApi {
    @GET("schedule")
    Call<ScheduleResponse> getSchedule(@Query("GroupId") int groupId);

    @GET("schedule")
    Call<ScheduleResponse> getScheduleByPersonId(@Query("PersonId") String personId);
}

File: app/src/main/java/com/example/irkpo_management/network/ScheduleApi.kt
package com.example.irkpo_management.network

import com.example.irkpo_management.models.ScheduleResponse
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface ScheduleApi {
    @GET("schedule")
    fun getSchedule(@Query("GroupId") groupId: Int): Call<ScheduleResponse?>?

    @GET("schedule")
    fun getScheduleByPersonId(@Query("PersonId") personId: String?): Call<ScheduleResponse?>?
}
